scripts/linters/js_ts_linter.py

```diff
@@ -1082,6 +1082,28 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.files_to_lint
 
+    def _get_trimmed_summary_message(self, lint_messages):
+        """Remove extra bits from eslint messages.
+
+        Args:
+            lint_messages: str. Messages returned by the js_ts linter.
+
+        Returns:
+            str. A string with the trimmed messages.
+        """
+        summary_messages = ''
+        # Extracting messages.
+        messages = lint_messages[0].split('\n')[:-4]
+        for message in messages:
+            if re.search(r'^\d+:\d+', message.lstrip()):
```

Comment:
No, both are different.

Reply:
Yeah, they are the same and I think duplicacy of just 4 lines is normal. You are the one who suggested to move all these in different modules.

Reply:
Yeah because it wasn't making sense there and it was hard to understand which output was coming from which linter now it's way better, right? 

Over here I was thinking to write a common method _trim_cross_unicode_from_message and use it in both the places. What do you think?